lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1922,6 +1912,16 @@ export class UserAgentApplication {
         return this.config.auth.redirectUri;
     }
 
+    /**
+     * Use to get the current url without the hash.
+     * This will be compared to request url to determine if redirect is needed
+     *
+     * @returns {string} current URL
+     */
+    public getCurrentUrl(): string {
```

Comment:
Can this go somewhere else other than `UserAgentApplication`? e.g. `UrlUtils`. We would like to have a few functions as possible here. And then add tests :)

Reply:
There was already an existing function in `UrlUtils` that does the same thing so I just renamed the existing function to be more usage agnostic and removed this function from `UserAgentApplication`